docs/_scripts/create_example.sh

```diff
@@ -4,9 +4,10 @@ echo "Create a text file and upload it to Pulp"
 
 echo 'Hello world!' > example.txt
 
-export ARTIFACT_HREF=$(http --form POST http://localhost/pulp/api/v3/artifacts/ \
+ARTIFACT_HREF=$(http --form POST http://localhost/pulp/api/v3/artifacts/ \
     file@./example.txt \
     | jq -r '.pulp_href')
 
 echo "Inspecting new artifact."
 http $BASE_ADDR$ARTIFACT_HREF
+export ARTIFACT_HREF
```

Comment:
all those scripts are sourced in the same shell process, so no need for any exported variables, afaics.

And given these snippets are presented verbatim in the docs, we should be as concise as possible.